### DISCUSSION TITLE
Upgrading from 8.0 CR7 => CR8

### DISCUSSION THREAD

**Discussion by MagnaFrisia90** [HCL-TECH-SOFTWARE/connections-automation#347] [Q&A] [answered]
Hello,
now that CR8 is released I was wondering if there is a playbook/guide available for upgrading my environment to CR8 using the automation framework ?
Sadly I can only find a "cfix" but not a "CR"-playbook which would also cover upgrading WAS to FP26, Db2 to MR9 & maybe also move from SDS 7.2 to SVD 10 .

Regards,
Jan

**Comment by MagnaFrisia90** [comment]
@sabrina-yee Hi ! Could you comment on if the automation framework also contains CR-upgrade capabilities or if you should always just perform a fresh install with the newest CR on a new VM and migrate data ?

**Reply by sabrina-yee** [reply, marked as answer]
Hi @MagnaFrisia90, apologies for my late reply as I was away.  
If the 8.0 CR7 deployment was installed using this automation framework, it can be upgraded to CR8 by setting `cnx_fixes_version` and `cnx_fixes_files` in `all.yml` and running the `playbooks/hcl/setup-connections-only.yml` playbook.  It will try to find the CR8 install kit in `cnx_fixes_repository_url` for the installation.  
https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/464bc47813342ffa6061f786aab05b735a7e0b70/environments/examples/cnx8/quick_start/group_vars/all.yml#L52C1-L54 

After the upgrade, it's recommended to check if the additional settings and customizations need to be reapplied if necessary.  For example, if anonymous access was disabled, the `playbooks/hcl/connections-restrict-access.yml` playbook should be run again.  If Docs is installed, please confirm if this [workaround](https://github.com/HCL-TECH-SOFTWARE/connections-automation/blob/main/documentation/FAQ.md#docs-thumbnails-stop-working-after-upgrading-connections-how-to-fix-that) is needed as well.

Hope this helps.
